stable/database/README.md

```diff
@@ -315,17 +319,15 @@ Verify the Helm chart:
 
 ```bash
 helm install nuodb/database -n database \
-    --set sm.persistence.enabled=true \
-    --set sm.persistence.storageClass=nuodb-archive \
```

Comment:
was this just wrong?

Reply:
Removed as I think we removed optional non-persistence long ago, cannot see any references to sm.persistence.enabled